services/workflows-service/src/statistics/statistics.controller.internal.ts

```diff
@@ -0,0 +1,22 @@
+import * as common from '@nestjs/common';
+import * as swagger from '@nestjs/swagger';
+import { ProjectIds } from '@/common/decorators/project-ids.decorator';
+import type { TProjectIds } from '@/types';
+import { StatisticsService } from '@/statistics/statistics.service';
+import { StatisticsOutputDto } from '@/statistics/statistics-output.dto';
+import { StatisticsInputDto } from '@/statistics/statistics-input.dto';
+
+@swagger.ApiExcludeController()
+@common.Controller('internal/statistics')
+export class StatisticsControllerInternal {
+  constructor(protected readonly service: StatisticsService) {}
+
+  @common.Post('')
+  @swagger.ApiOkResponse({ type: StatisticsOutputDto })
```

Comment:
New endpoints should use `@ApiResponse` and `@Validate` with Typebox like its done in the event endpoint and the new ones for editing workflow definitions. 🙂